platform/web/js/libs/library_godot_audio.js

```diff
@@ -858,7 +858,10 @@ class Bus {
 	 * @returns {void}
 	 */
 	setVolumeDb(val) {
-		this._gainNode.gain.value = GodotAudio.db_to_linear(val);
+		const linear = GodotAudio.db_to_linear(val);
+		if (isFinite(linear)) {
+			this._gainNode.gain.value = linear;
+		}
```

Comment:
```suggestion
		} else {
			this._gainNode.gain.value = 0;
		}
```
If we want uniformity between platforms, probably unnecessary though but would silence when fed invalid input as is the case otherwise

Reply:
I would rather test directly with `isNaN()` rather than `isFinite()`, because `-Infinity` is a valid dB value.

Reply:
But this isn't a dB value, it's a linear value

Reply:
I don't understand your edit, as I tested locally and `db_to_linear(Infinity)` returns `Infinity`.

What I suggest is that we return early if `isNaN(val)` returns `true`.

Reply:
Yes but please check the issue, it will error out, the issue is that you can't assign a non-finite value, your suggested fix isn't valid

This is a potential mini suggestion but isn't really relevant, the issue is that we can't assign a non-finite value to the gain